packages/api/models/collections.js

```diff
@@ -64,6 +66,56 @@ class Collection extends Manager {
 
     return super.create(item);
   }
+
+  /**
+   * Delete a collection
+   *
+   * @param {string} name - the collection name
+   * @param {string} version - the collection version
+   */
+  async delete(name, version) {
+    if (!(await this.exists(name, version))) throw new Error('Collection does not exist');
+
+    const associatedRuleNames = (await this.associatedRules(name, version))
```

Comment:
Is the return from `this.associatedRules` not already an array? If so, it seems like maybe mapping it to an array should happen within that function

Reply:
The return value of `this.associatedRules` is an array of rule objects.  These two lines are taking that array of rules objects and converting it into an array of rule names.

Reply:
OK. The mapping to an array of rule names is probably fine as is then